Automation/SQL/azureDeploy.ps1

```diff
@@ -31,14 +35,15 @@ if (!$resourceGroupExists) {
         New-AzResourceGroup -Name $resourceGroupName -Location "$location" -Force
         Write-Host "Created resource group $resourceGroupName"
 } else {
-        Write-Host "Resource group already exists."
+        Write-Host "Resource group $resourceGroupName already exists."
 }
 
 # Deploy template
-Write-host "Creating primary server..."
+Write-host "Creating primary server and db..."
 New-AzResourceGroupDeployment -ResourceGroupName $resourceGroupName `
 -TemplateFile $pathToAzSqlTemplate -administratorLogin $administratorLogin `
--administratorLoginPassword $administratorLoginPassword
+-administratorLoginPassword $administratorLoginPassword -serverName $serverName `
+-sqlDBName $sqlDBName -location $location -createdBy $createdBy -creatorsEmail $creatorsEmail
```

Comment:
Same here - I don't think you need to have parameters for the sqlDBName or the serverName at all, because you're building those in the param files. If you want there to be the possibility to  provide the input, then you might do a conditional here to test if those variables are null, and only pass them in if they're not.

Reply:
thank you for the comment! It's a bit tricky here - the params are dynamic and take in the location as one of the building blocks. The location is `westus2`, but Toddy wants us to use usw2 in our naming conventions, so I decided to introduce the parameters for the sqlDBName and the serverName to have the prescribed naming convention

Reply:
Oh I see! 
Well hm, does location need to be a parameter then, if it's always westus2?
Could you just build the strings in the .ps1 file instead of the .json file?